pom.xml

```diff
@@ -43,32 +43,32 @@
   <build>
     <pluginManagement>
       <plugins>
-        <plugin>
-          <groupId>org.apache.felix</groupId>
-          <artifactId>maven-bundle-plugin</artifactId>
-          <version>2.3.7</version>
-          <extensions>true</extensions>
-          <executions>
-            <execution>
-              <phase>package</phase>
-              <goals>
-                <goal>bundle</goal>
-              </goals>
-            </execution>
-          </executions>
-          <configuration>
-            <archive>
-              <manifest>
-                <mainClass>${app.main.class}</mainClass>
-              </manifest>
-            </archive>
-            <instructions>
-              <Embed-Dependency>*;inline=false;scope=compile</Embed-Dependency>
-              <Embed-Directory>lib</Embed-Directory>
-              <Embed-Transitive>true</Embed-Transitive>
-            </instructions>
-          </configuration>
-        </plugin>
+        <!--<plugin>-->
```

Comment:
Do you think it's worth using a block comment here? Additionally, would be good to include a small comment about why this is being commented out. Applies both to this block (build -> pluginManagement -> plugins -> plugin)  as well as the block below (build -> plugins -> plugin)